docs/cudf/source/PandasCompat.rst

```diff
@@ -0,0 +1,4 @@
+Pandas Compatibility Notes
+==========================
+
+.. pandas-compat-list::
```

Comment:
I believe the trailing `::` is redundant in this case since you're not passing any arguments, but it's probably fine to leave it as is unless you're in a position to test and remove fairly quickly.